### PR TITLE
Add Senses's WEIZEN board from Senses IoT platform

### PR DESCRIPTION
**Pull request by Isaranu**
Please add our product "Senses's WEIZEN" (ESP32 series) into boards.txt
Thank you very much.
(Re-PR again)

### CODE REVIEW

File: boards.txt [hunk 1]
```diff
@@ -4774,3 +4774,45 @@ metro_esp-32.menu.DebugLevel.verbose=Verbose
...
+sensesiot_weizen.upload.maximum_size=1310720
+sensesiot_weizen.upload.maximum_data_size=327680
+sensesiot_weizen.upload.wait_for_upload_port=true
+
+sensesiot_weizen.serial.disableDTR=true
+sensesiot_weizen.serial.disableRTS=true
+
+sensesiot_weizen.build.mcu=esp32
+sensesiot_weizen.build.core=esp32
+sensesiot_weizen.build.variant=sensesiot_weizen
```
Comment (me-no-dev) [inline boards.txt]: you did not add a header for the variant. If this is same as ESP32 dev boards, change the variant to "esp32", else you need to add the "pins_arduino.h" in the proper variant subfolder

Comment (Isaranu) [reply boards.txt]: OK, I see.

Comment (Isaranu) [reply boards.txt]: I updated it already, please verify.
Thank you.

**Comment by me-no-dev** [comment]
yup :) all good now.

**Comment by Isaranu** [comment]
When I can see our board in Arduino IDE board's list ?

**Pull request merged**